kayenta-oss/src/main/java/com/netflix/kayenta/oss/config/OssConfiguration.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 Alibaba Group.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.kayenta.oss.config;
```

Comment:
Can you please change the name of the package and of the module to something more specific to Alibaba than simply `oss`? I think the `oss` name will be confusing for future readers.

Reply:
Looks like @fieldju already made a similar comment, so will resolve this one.